ios/Classes/NativeBridge/Handlers/RegistrationHandler.swift

```diff
@@ -167,20 +167,22 @@ extension RegistrationHandler : RegistrationConnectorToHandlerProtocol {
         presentBrowserUserRegistrationView(registrationUserURL: url, webSignInType: webSignInType)
     }
 
+    // function only used for testing
```

Comment:
These can actually be deleted, same for the tests, and `handleCustomRegistrationAction` function should also be deleted. `sendCustomOtpNotification` should also  be deleted

Reply:
done